.github/workflows/ci.yml

```diff
@@ -148,7 +148,11 @@ jobs:
 
 
   pass:
+    if: always()
     needs: [tests, tests-pypy, dist]
     runs-on: ubuntu-latest
     steps:
-      - run: echo "All jobs passed"
+     - name: Decide whether the needed jobs succeeded or failed
```

Comment:
I often use prettier, and just noticed that I didn’t have it here this morning when preparing to teach a winter session on code quality assurance tooling. :)

I’ve heard a bit about yamllint, can try it. I like 0-space, but prettier kind of forces 2-space.  I like autofixers though, especially now I’m starting to get spoiled with Ruff.

I want to check its performance, too, I recently helped Awkward move to Ruff, taking the pre-commit time from 12 seconds to 6 seconds, and now the slowest step is the check-YAML pre-commit step. 🤦

Reply:
The json schema one is new to me, interesting!

Reply:
I like 0-indent YAML sequences too.
Jsonschema is nice since it integrates into code editors, and is not just a linter. That specific thingy uses a community-maintained schema for GH workflows but you can write your own schemas, that might be specific to your projects.